pressbooks/pressbooks-book
# Fix jQuery deprecation warnings

This is a partial fix for #751 

This PR tries to remove all deprecation warnings by replacing deprecated calls with the proper approach.
This also updates the test matrix to include WP 5.7.2.

#### How to test

1. Navigate through a book and perform all kinds regular actions
2. Click on hide/display content buttons
3. Resize the page

None of those actions should trigger `JQMIGRATE: ... is deprecated`
